bip-0141.mediawiki

```diff
@@ -118,7 +118,7 @@ Sigops in the current pubkey script, signature script, and P2SH check script are
 The sigop limit is likewise quadrupled to ≤ 80,000.
 
 In addition, opcodes within the witness program are counted identical to as previously within the P2SH check script.
-That is, CHECKSIG is counted as only 1 sigop in a witness program, despite P2SH check script's CHECKSIG becoming 4 sigops with the forementioned change.
+That is, CHECKSIG in a witness program is counted as only 1 sigop, and m-of-n CHECKMULTISIG in a witness program is counted as n sigop. This rule applies to both native witness program and P2SH witness program.
```

Comment:
it sounds a bit cumbersome, as it already mentions "is counted as only 1 sigop".

I also want to make it clear that witness program in P2SH is also counted without quadrupling.